model/src/main/java/com/cloudera/nav/sdk/model/entities/Entity.java

```diff
@@ -344,4 +351,10 @@ public boolean equals(Object o) {
   public int hashCode() {
     return identity.hashCode();
   }
+
+  public Boolean getIsEndPoint() { return isEndPoint; }
+
+  public void setIsEndPoint(Boolean isEndPoint) {
```

Comment:
sure. Lets discuss about this tomorrow. We need a way to differentiate when to write certain entities to the json and when to not write them. The EndPointProxy achieved this same purpose.